lib/tests/azure/test_config.py

```diff
@@ -38,8 +38,8 @@ def test_azure_cluster():
         "f68a6145029a9ea843b0305c90a7f5f0334d8a8ceeea94734267ec36421fe7fe"
     )
     assert c.checksum("accumulo") == (
-        "sha256:"
-        "df172111698c7a73aa031de09bd5589263a6b824482fbb9b4f0440a16602ed47"
+        "sha512:"
+        "b443839443a9f5098b55bc5c54be10c11fedbaea554ee6cd42eaa9311068c70bd611d7fc67698c91ec73da0e85b9907aa72b98d5eb4d49ea3a5d51b0c6c5785f"
```

Comment:
Just split the string into 2 strings. Python string concatenation will do the rest.

```suggestion
        "b443839443a9f5098b55bc5c54be10c11fedbaea554ee6cd42eaa9311068c70b"
        "d611d7fc67698c91ec73da0e85b9907aa72b98d5eb4d49ea3a5d51b0c6c5785f"
```

Reply:
Splitting into two strings makes it slightly harder to compare with the upstream published checksum, but if that's the only way to get around it, that's fine with me. I was hoping for the ability to add an exception to the check. Do you know if that is possible?